actions/dump-charm-artifacts/action.yml

```diff
@@ -0,0 +1,52 @@
+name: 'Dump Charm Log Artifacts'
+description: 'Dumps to Artifacts useful Juju, Kubernetes, and Charmcraft log information.  Requires kubectl to be set up with a context, and charmcraft to be installed.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Check out repo
+      uses: actions/checkout@v3
+
+    # Setup
+
+    - name: Create artifact collection directory
+      id: mkdir
+      shell: bash
```

Comment:
I wonder if there is a way to set this globally. I could not find anything, but maybe you know?

Reply:
do you mean the `shell` globally?  That would be really nice.  I see [this unanswered request](https://github.com/orgs/community/discussions/46670) for that, so I think no :(

Reply:
Yeah, the `shell`. So sad it's not an option. :(